src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -107,4 +107,8 @@ public boolean isAtTargetSpeed() {
         double currentRPM = this.shooterEncoder.getVelocity();
         return (Math.abs(currentRPM - targetRPM) <= 100);
     }
+
+    public int getDistanceToRPM(int distance) {
+        return (int)((distance*(-1.05469))+4432.2);
```

Comment:
Don't use magic numbers. Where did these come from?!

Reply:
We ran the shooter at different distances to make a linear equation of the needed RPM

Reply:
Maybe variables `speedCoefficient` and `speedOffset`